code/modules/projectiles/guns/energy/special.dm

```diff
@@ -3,6 +3,8 @@
 	desc = "A man-portable anti-armor weapon designed to disable mechanical threats at range."
 	icon_state = "ionrifle"
 	inhand_icon_state = null	//so the human update icon uses the icon_state instead.
+	worn_icon_state = null
+	worn_icon = 'icons/mob/clothing/back.dmi'

```

Comment:
this is already handled by `build_worn_icon` via the slot flag, remove it.

Reply:
Reverted. Had the worn_icon set to make the other slots use a working icon. I'll add it to the list of missing textures.